catalog/format/iceberg/src/main/java/org/projectnessie/catalog/formats/iceberg/nessie/NessieModelIceberg.java

```diff
@@ -755,7 +757,33 @@ public static long safeUnbox(Long value, long defaultValue) {
     return value != null ? value : defaultValue;
   }
 
-  public static NessieTableSnapshot newIcebergTableSnapshot(List<IcebergMetadataUpdate> updates) {
+  public static String defaultIcebergLocation(String warehouseLocation, ContentKey key) {
+    String baseLocation = warehouseLocation;
```

Comment:
If you prefer to do this here, fine with me. You tell me :-)

Reply:
Maybe flat structure (no namespace) in this PR, plus more advanced strategies as a follow-up?

Reply:
What do you mean by "object store naming strategy", is it whether to use path-style or virtual-host style?

Reply:
Nah - with "naming strategy" I meant the "randomized prefix" from `o.a.iLocationProviders.ObjectStoreLocationProvider` - but it seems my brain was somewhere else, because this change is about the base location - the the actual object location. Please disregard my comment and ignore everything I said or recommended above.